### PR TITLE
chore: add PR template

### PR DESCRIPTION
**Pull request by jtmalinowski**

### CODE REVIEW

**Review by mhennoch — APPROVED** [review]

**Review by t2t2 — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Description
+
+_Replace this with a description of the PR. Explain the problem it solves and the proposed solution._
+
+_List Github/JIRA issue(s) this PR fixes._
```
Comment (theletterf) [inline .github/pull_request_template.md:5]: Not sure Jira issues should go here if it's a public repo.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Type of change
+
+Please delete options that are not relevant.
```
Comment (theletterf) [inline .github/pull_request_template.md:9]:
```suggestion
Delete options that are not relevant.
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- Chore or internal change (changes not visible to the consumers of the package)
+- This change requires a documentation update
```
Comment (theletterf) [inline .github/pull_request_template.md:15]:
```suggestion
- Documentation update
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+# How Has This Been Tested?
```
Comment (theletterf) [inline .github/pull_request_template.md:17]:
```suggestion
# How has this been tested?
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- [ ] Tested manually
```
Comment (theletterf) [inline .github/pull_request_template.md:19]:
```suggestion
- [ ] Manual testing
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] Added unit tests
+- [ ] Added integration tests
+
+# Checklist:
+
+- [ ] Changelogs have been updated
+- [ ] Unit tests have been added/updated
+- [ ] Documentation has been updated
+- [ ] Team's resident editor is tagged on the PR (if there are changes to the documentation)
```
Comment (theletterf) [inline .github/pull_request_template.md:28]:
```suggestion
- [ ] Team's resident technical writer tagged on the PR (if there are changes to the documentation)
```

**Review by theletterf — CHANGES_REQUESTED** [review]

**Pull request merged**